admin_guide/scheduling/pod_placement.adoc

```diff
@@ -0,0 +1,380 @@
+pods.
+
+The Pod Node Constraints admission controller ensures that pods 
+are deployed onto only specified node hosts using labels]
+and prevents users without a specific role from using the 
+`nodeSelector` field to schedule pods.
+
+[[constraining-pod-placement-labels]]
+== Constraining Pod Placement Using Node Name
+
+Use the Pod Node Constraints admission controller to ensure a pod 
+is deployed onto only a specified node host by assigning it a label
+and specifying this in the `nodeName` setting in a pod configuration.
+
+. Ensure you have the desired labels
+ifdef::openshift-enterprise,openshift-origin[]
+(see xref:../../admin_guide/manage_nodes.adoc#updating-labels-on-nodes[Updating
+Labels on Nodes] for details)
+endif::openshift-enterprise,openshift-origin[]
+ifdef::openshift-dedicated[]
+(request changes by opening a support case on the
+https://access.redhat.com/support/[Red Hat Customer Portal])
+endif::openshift-dedicated[]
+and xref:../../admin_guide/managing_projects.adoc#using-node-selectors[node selector]
+set up in your environment.
++
+For example, make sure that your pod configuration features the `nodeName`
+value indicating the desired label:
++
+----
+apiVersion: v1
+kind: Pod
+spec:
+  nodeName: <value>
+----
+
+. Modify the master configuration file
+(*_/etc/origin/master/master-config.yaml_*) in two places:
++
+.. Add `PodNodeConstraints` to the `admissionConfig` section:
++
+----
+...
+admissionConfig:
+  pluginConfig:
+    PodNodeConstraints:
+      configuration:
+        apiversion: v1
+        kind: PodNodeConstraintsConfig
+...
+----
+
+.. Then, add the same to the `kubernetesMasterConfig` section:
++
+----
+...
+kubernetesMasterConfig:
+  admissionConfig:
+    pluginConfig:
+      PodNodeConstraints:
+        configuration:
+          apiVersion: v1
+          kind: PodNodeConstraintsConfig
+...
+----
+
+. Restart {product-title} for the changes to take effect.
+ifdef::openshift-origin[]
++
+----
+# systemctl restart origin-master
+----
+endif::[]
+ifdef::openshift-enterprise[]
++
+----
+# systemctl restart atomic-openshift-master
+----
+endif::[]
+
+
+[[constraining-pod-placement-nodeselector]]
+== Constraining Pod Placement Using a Node Selector
+
+Using xref:../../admin_guide/managing_projects.adoc#using-node-selectors[node selectors], 
+you can ensure that pods are only placed onto nodes with specific labels. As a cluster administrator, you can
+use the Pod Node Constraints admission controller to set a policy that prevents users without the *pods/binding* permission 
+from using node selectors to schedule pods.
+
+The `nodeSelectorLabelBlacklist` field of a master configuration file gives you
+control over the labels that certain roles can specify in a pod configuration's
+`nodeSelector` field. Users, service accounts, and groups that have the
+*pods/binding* permission xref:../../admin_guide/manage_rbac.adoc#admin-guide-manage-rbac[role]
```

Comment:
@mburke5678 - manage_rbac.adoc was introduced in 3.7, not 3.6. Before that it was manage_users (from memory). This needed to point to that file instead of this one.